pyproject.toml

```diff
@@ -11,6 +11,8 @@ dynamic = ["version"]
 dependencies = [
     "plantcv",
     "matplotlib",
+    "jupyterlab",
```

Comment:
jupyterlab can be removed as a dependency as it's not directly needed to run the code

Reply:
Sure. Thanks.